test/e2e-tests/README.md

```diff
@@ -16,6 +16,8 @@ The following enviroment variables are required to run the test setup and the te
 - `S3_ENDPOINT` - Endpint of the bucket
 - `IMAGE_REGISTRY_USERNAME` - quay.io username
 - `IMAGE_REGISTRY_PASSWORD` - quay.io password
+- `GIT_TOKEN` - Auth token used by the git ops pipeline to make a pull request, [see info here](https://github.com/jackdelahunt/ai-edge/tree/git-ops-e2e-tests/pipelines#git-repository-and-credentials)
```

Comment:
```suggestion
- `GIT_TOKEN` - Auth token used by the git ops pipeline to make a pull request, [see info here](./pipelines#git-repository-and-credentials)
```

I _think_ that should work, without needing to refer to a temporary branch on your fork.

Reply:
I don't know why I linked my fork, fixed now